plugins/CustomDimensions/vue/src/Edit/Edit.vue

```diff
@@ -0,0 +1,350 @@
+              :inline-help="translate('CustomDimensions_CannotBeDeleted')"
+            >
+            </Field>
+          </div>
+          <div
+            class="row form-group"
+            v-show="doesScopeSupportExtraction"
+          >
+            <h3 class="col s12">{{ translate('CustomDimensions_ExtractValue') }}</h3>
+            <div class="col s12 m6">
+              <div
+                v-for="(extraction, index) in dimension.extractions"
+                :class="`${index}extraction `"
+                :key="index"
+              >
+                <div class="row">
+                  <div class="col s12 m6">
+                    <div>
+                      <Field
+                        uicontrol="select"
+                        :name="`${index}dimension`"
+                        v-model="extraction.dimension"
+                        :full-width="true"
+                        :options="extractionDimensionsOptions"
+                      >
+                      </Field>
+                    </div>
+                  </div>
+                  <div class="col s12 m6">
+                    <div>
+                      <Field
+                        uicontrol="text"
+                        :name="`${index}pattern`"
+                        v-model="extraction.pattern"
+                        :full-width="true"
+                        :title="extraction.dimension === 'urlparam'
+                          ? 'url query string parameter'
+                          : 'eg. /blog/(.*)/'"
+                      >
+                      </Field>
+                    </div>
+                  </div>
+                  <div class="col s12">
+                    <span
+                      class="icon-plus"
+                      v-show="extraction.pattern"
+                      @click="addExtraction()"
+                    />
+                    <span
+                      class="icon-minus"
+                      v-show="dimension.extractions.length > 1"
+                      @click="removeExtraction(index)"
+                    />
+                  </div>
+                </div>
+              </div>
+              <div class="row">
+                <div class="col s12">
+                  <div>
+                    <Field
+                      uicontrol="checkbox"
+                      name="casesensitive"
+                      v-show="dimension.extractions[0]?.pattern"
+                      v-model="dimension.case_sensitive"
+                      :title="translate('Goals_CaseSensitive')"
+                    >
+                    </Field>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="col s12 m6 form-help">
+              {{ translate('CustomDimensions_ExtractionsHelp') }}
+            </div>
+          </div>
+          <input
+            class="btn update"
+            type="submit"
+            value="Update"
```

Comment:
Not sure if that was the case before. But would be nice to have this button and the following one translatable.